### ISSUE TITLE
aria-describedby when supplied with multiple ids, doesn't read in order( JAWS + Google)

### ISSUE THREAD

**Issue by ahsansalim200** [FreedomScientific/VFO-standards-support#496]
Use the test html I am attaching to reproduce the issue. 

```html
<html>
    <body>
        <style type="text/css">
            .tg  {border-collapse:collapse;border-spacing:0;}
            .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
              overflow:hidden;padding:10px 5px;word-break:normal;}
            .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
              font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
            .tg .tg-0lax{text-align:left;vertical-align:top}
            </style>
            <table class="tg">
            <thead>
              <tr>
                <th class="tg-0lax" id="0_0">This</th>
                <th class="tg-0lax" id="1_0">issue is</th>
                <th class="tg-0lax" id="2_0">one third</th>
                <th class="tg-0lax" id="3_0">one fourth</th>
                <th class="tg-0lax" id="4_0">one fifth</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td class="tg-0lax" id="0_1">not present</td>
                <td class="tg-0lax" id="1_1">in other</td>
                <td class="tg-0lax" id="2_1">Two third</td>
                <td class="tg-0lax" id="3_1">Two fourth</td>
                <td class="tg-0lax" id="4_1">Two fifth</td>
              </tr>
              <tr>
                <td class="tg-0lax">browsers</td>
                <td class="tg-0lax">and also </td>
                <td class="tg-0lax">three first</td>
                <td class="tg-0lax">three first</td>
                <td class="tg-0lax">three first</td>
              </tr>
              <tr>
                <td class="tg-0lax">older versions</td>
                <td class="tg-0lax"><div><span>
                  of jaws like
                </span></div></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
              </tr>
              <tr>
                <td class="tg-0lax">version 18</td>
                <td class="tg-0lax">and below</td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
              </tr>
              <tr>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
              </tr>
              <tr>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"></td>
              </tr>
              <tr>
                <td class="tg-0lax"></td>
                <td class="tg-0lax"> <button  aria-describedby="0_0 0_1 1_0 1_1 0_2 1_2 0_3 1_3 0_4 1_4"> two </button></td>
                <td class="tg-0lax"> <button> three </button></td>
                <td class="tg-0lax"> <button> four </button></td>
                <td class="tg-0lax"><button> five </button> </td>
              </tr>
            </tbody>
            </table>
    </body>
</html>
```

Tab on to the first button and listen to the order in which the text is being read out. It is not reading out in the order in which the ids in aria-describedby is specified.

Chrome Version : 88.0.4324.150
Jaws version: 2021.2012.57 ILM

**Comment by JAWS-test** [comment]
I can confirm the problem. It's very strange because Chrome correctly submits the description to the Accessibility API, but JAWS doesn't seem to use it.

The problem does not occur with NVDA+Chrome and not with JAWS+Firefox.

**Comment by StephenBeckettTPGi** [comment]
Also can confirm this behaviour in Chrome/JAWS 21 - changing the ids, removing some ids, shuffling them about, results in various incorrect orders being read out. Eg. the following markup results in "In other This issue is not present" rather than "This not present issue is in other". JAWS/FF reads out order correctly.

```
<table class="tg">
  <thead>
    <tr>
      <th class="tg-0lax" id="a">This</th>
      <th class="tg-0lax" id="c">issue is</th>
      <th class="tg-0lax" id="2_0">one third</th>
      <th class="tg-0lax" id="3_0">one fourth</th>
      <th class="tg-0lax" id="4_0">one fifth</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td class="tg-0lax" id="b">not present</td>
      <td class="tg-0lax" id="d">in other</td>
      <td class="tg-0lax" id="2_1">Two third</td>
      <td class="tg-0lax" id="3_1">Two fourth</td>
      <td class="tg-0lax" id="4_1">Two fifth</td>
    </tr>
    <tr>
      <td class="tg-0lax"></td>
      <td class="tg-0lax"><button aria-describedby="a b c d">two</button></td>
      <td class="tg-0lax"><button>three</button></td>
      <td class="tg-0lax"><button>four</button></td>
      <td class="tg-0lax"><button>five</button></td>
    </tr>
  </tbody>
</table>
```

Even this is not read out in correct order.

```
<!DOCTYPE html>

<html>
<body>
  <p id="a">This</p>
  <p id="b">issue is</p>
  <p id="c">not present</p>
  <p id="d">in other</p>

  <button aria-describedby="a b c d">two</button>
</body>

</html>
```

**Comment by BenKeyFSI** [comment]
Reported as Bug 123396 {When multiple elements are referenced by the aria-describedby attribute the items are not read in the expected order when JAWS speaks the element description} in the Vispero Bugzilla.

**Comment by stevefaulkner** [comment]
Tested, still present in JAW 2023.144 with Chrome latest
not present in Firefox 114.0b5 (64-bit)
test case: https://codepen.io/stevef/pen/zYMYwGP

**Comment by stevefaulkner** [comment]
Marked as Fixed in internal bug tracker
